src/target/common/devo/Makefile.inc

```diff
@@ -51,6 +51,11 @@ LFLAGS2OPT  = -Wl,-T$(SDIR)/target/$(TARGET)/$(TARGET)_opt.ld
 # Enable warning as error
 CFLAGS += -Werror
 
+# Enable stack usage analyze
+ifeq "$(TYPE)" "dev"
```

Comment:
Might be better to have a new TYPE here.  The dev type enables some debugging changes like printf output to serial port that could affect stack measurements.